src/node/src/command.rs

```diff
@@ -126,6 +127,18 @@ pub enum DB3Command {
         public_grpc_url: String,
     },
 
+    /// Start db3 indexer
+    #[clap(name = "indexer")]
+    Indexer {
+        /// the url of db3 grpc api
+        #[clap(long = "url", global = true, default_value = "http://127.0.0.1:26659")]
```

Comment:
the  port 26659 has been used

Reply:
this is used to connect db3 node to sync/fetch block from db3

Reply:
change argument to db3_storage_grpc_url